api/nodes/views.py

```diff
@@ -27,7 +30,6 @@ def get_node(self):
         self.check_object_permissions(self.request, obj)
         return obj
 
-
```

Comment:
Class definitions should be surrounded by two blank lines. This happens a few others places in this file as well.

ref: https://www.python.org/dev/peps/pep-0008/#blank-lines